appveyor.yml

```diff
@@ -67,11 +66,14 @@ before_build:
 
 test:
   assemblies:
-    - RiotSharpTest.dll
+    - RiotSharp.Test.dll
+
+test_script:
+  - cmd: dotnet test c:\projects\RiotSharp\RiotSharp.Test\RiotSharp.Test.csproj
 
 # code coverage
 after_test:
-  - .\packages\OpenCover.4.6.519\tools\OpenCover.Console.exe -register:user -target:"c:\Program Files (x86)\Microsoft Visual Studio 14.0\Common7\IDE\MSTest.exe" -targetargs:"/noresults /noisolation /testcontainer:""c:\projects\RiotSharp\RiotSharpTest\bin\Release\RiotSharpTest.dll" -filter:"+[RiotSharp]*  -[RiotSharp]RiotSharp.*Endpoint.*" -excludebyattribute:*.ExcludeFromCodeCoverage* -hideskipped:All -output:.\RiotSharp_coverage.xml
+  - .\packages\OpenCover.4.6.519\tools\OpenCover.Console.exe -register:user -target:"c:\Program Files (x86)\Microsoft Visual Studio 14.0\Common7\IDE\MSTest.exe" -targetargs:"/noresults /noisolation /testcontainer:""c:\projects\RiotSharp\RiotSharp.Test\bin\Release\netcoreapp1.1\RiotSharp.Test.dll" -filter:"+[RiotSharp]*  -[RiotSharp]RiotSharp.*Endpoint.*" -excludebyattribute:*.ExcludeFromCodeCoverage* -hideskipped:All -output:.\RiotSharp_coverage.xml
```

Comment:
I know. I'm gonna fix it later, but for now it is more important to merge this into develop.